pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
So just what exactly is `downcast` supposed to take as a parameter?  It seems like it can only take three different strings:

`"int"`, `"unsigned"`, and `"float"` because other `dtypes` (e.g. `np.int8`) make less sense in the context you are providing.  Or can they take those as well and implement the behaviour I currently have in this PR?